dffml/model/model.py

```diff
@@ -87,6 +98,85 @@ def __call__(self) -> ModelContext:
         self._make_config_location()
         return self.CONTEXT(self)
 
+    async def __aenter__(self):
+        if any(
+            [
+                self.config.location.is_file(),
+                get_archive_path_info(self.config.location)[0]
+                in ["zip", "tar"],
+            ]
+        ):
+            # TODO: Config can be read from a file named config.json
+            # if it exists and self.config.location can be overridden
+            # with the loaded config [depends on PR intel/dffml#1122]
+            temp_dir = self._get_directory()
+            self.location = temp_dir
+
+        if self.config.location.is_file():
+            load_flow = getattr(self.config, "location_load", None)
+            await self._run_operation(
+                self.config.location, temp_dir, load_flow
+            )
+            # Load values from config if it exists
+            config_path = self.temp_dir / "config.json"
+            if config_path.exists():
+                with open(config_path) as config_handle:
+                    loaded_config = json.load(config_handle)
+                    for prop, value in loaded_config.items():
+                        # TODO: Need to figure out how loading of
+                        # properties should be done.
+                        pass
```

Comment:
Let's leave this as a TODO for now, but within this loop body, instead of let's set anything that is of the correct type. Let's `self.logger.warning` and log if there is a differnce.

Reply:
Have added this code in a local commit, would push this soon :)

Reply:
Done 👍🏽